alphametics.json

```diff
@@ -21,6 +21,16 @@
                 "description": "solve puzzle with exponents",
                 "puzzle": "PI * R ^ 2 == AREA",
                 "expected": { "P": 9, "I": 6, "R": 7, "A": 4, "E": 0 }
+            },
+            {
+                "description": "invalid solution returns null",
```

Comment:
Ah, so what I hear from this is that there are things we wish to convey:

1. "This puzzle has no valid solutions - any solution to it must have the same value assigned to two letters"
2. "Any puzzle with no valid solutions should have an expectation of null."

I would note that point 2 is not specific to this puzzle or any specific puzzle at all; it could apply to any puzzle. So perhaps that is an indication it should go elsewhere.

Another thing to consider is that explicitly putting null in the description may throw off languages for which null doesn't exist (if their generators directly use the description without modification).

I am thinking a title of "solutions can't have the same digit assigned to multiple letters".